export/ios.go

```diff
@@ -0,0 +1,252 @@
+package export
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/bitrise-tools/go-xcode/certificateutil"
+	"github.com/bitrise-tools/go-xcode/profileutil"
+	glob "github.com/ryanuber/go-glob"
+)
+
+// IosCodeSignGroup ...
+type IosCodeSignGroup struct {
+	Certificate        certificateutil.CertificateInfoModel
+	BundleIDProfileMap map[string]profileutil.ProvisioningProfileInfoModel
+}
+
+// CreateIosCodeSignGroups ...
+func CreateIosCodeSignGroups(selectableGroups []SelectableCodeSignGroup) []IosCodeSignGroup {
+	alreadyUsedProfileUUIDMap := map[string]bool{}
+
+	singleWildcardGroups := []IosCodeSignGroup{}
+	xcodeManagedGroups := []IosCodeSignGroup{}
+	notXcodeManagedGroups := []IosCodeSignGroup{}
+	remainingGroups := []IosCodeSignGroup{}
+
+	for _, selectableGroup := range selectableGroups {
+		certificate := selectableGroup.Certificate
+		bundleIDProfilesMap := selectableGroup.BundleIDProfilesMap
+
+		bundleIDs := []string{}
+		profiles := []profileutil.ProvisioningProfileInfoModel{}
+		for bundleID, matchingProfiles := range bundleIDProfilesMap {
+			bundleIDs = append(bundleIDs, bundleID)
+			profiles = append(profiles, matchingProfiles...)
+		}
+
+		//
```

Comment:
It would be more readable with separate functions:
```go
func singleWildcardGroups(group SelectableCodesingGroup) []IosCodeSignGroup 
func xcodeManagedGroups(group SelectableCodesingGroup) []IosCodeSignGroup 
func notXcodeManagedGroups(group SelectableCodesingGroup) []IosCodeSignGroup 
func remainingGroups(group SelectableCodesingGroup) []IosCodeSignGroup 
```
Then filter for unique items.
Or use an extra parameter for the functions
```go
func singleWildcardGroups(group SelectableCodesingGroup, profileUUIDSet map[string]bool) []IosCodeSignGroup
```